README.md

```
@@ -77,6 +80,115 @@ pnpm add mermaid
 ```
 
+#### 降级行为说明
+
+X-Markdown 采用优雅降级策略，即使不安装可选依赖，组件也能正常工作：
+
+- **未安装 shiki/shiki-stream**：代码块以纯文本模式显示，无语法高亮
+- **未安装 mermaid**：Mermaid 图表以代码块形式显示原始内容
+
+当可选依赖缺失时，浏览器控制台会显示**一次**友好的提示信息：
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)